flink-table-store-core/src/main/java/org/apache/flink/table/store/file/schema/TableSchema.java

```diff
@@ -129,6 +134,33 @@ public Map<String, String> options() {
         return options;
     }
 
+    public List<String> bucketKeys() {
+        String key = options.get(BUCKET_KEY.key());
+        if (StringUtils.isNullOrWhitespaceOnly(key)) {
+            return Collections.emptyList();
+        }
+        List<String> bucketKeys = Arrays.asList(key.split(","));
```

Comment:
split and trim?

Reply:
Let it go, user can see the wrong field in the exception.